gateway/app/main/processes/controller.py

```diff
@@ -0,0 +1,59 @@
+from .federated_learning_process import FederatedLearningProcess
+
+
+class FLPController:
+    """ This class implements controller design pattern over the federated learning processes."""
+
+    def __init__(self):
+        self.processes = {}
+
+    def create_process(
+        self,
+        model,
+        client_plans,
+        client_protocols,
```

Comment:
since we are not planning to work with protocol for a while, and when protocols are ready it may not be 100% required, should we make this a default arg for ease of use? 
i.e. client_protocols=None,

Reply:
Yes, it makes a lot of sense, @Prtfw.